src/scitokens/utils/keycache.py

```diff
@@ -60,28 +61,28 @@ def getinstance():
             KEYCACHE_INSTANCE = KeyCache()
         return KEYCACHE_INSTANCE
 
-    def addkeyinfo(self, issuer, key_id, public_key):
+    def addkeyinfo(self, issuer, key_id, public_key, cache_timer=0):
```

Comment:
Can you document the parameter meaning and types?

Particularly, it's not clear what unit `cache_timer` should be specified in.